package.json

```diff
@@ -1,17 +1,15 @@
 {
   "scripts": {
     "kusama:clean": "shx rm -rf node_modules/.cache; docusaurus clear kusama-guide",
-    "kusama:start": "yarn fetch --isPolkadot=false; docusaurus start kusama-guide",
-    "prekusama:build": "npm-run-all -ln kusama:clean -p \"fetch --isPolkadot=false\" kusama:copy-assets --aggregate-output",
```

Comment:
As far as I could tell these `pre:build` commands aren't used but please let me know if this is not the case.